packages/flame/lib/src/sprite_batch.dart

```diff
@@ -170,29 +171,66 @@ class SpriteBatch {
     RSTransform? defaultTransform,
     Images? images,
     bool useAtlas = true,
+    bool needsFlips = true,
```

Comment:
This should be false by default, because we can detect when the user needs to use this option but forgot to set it, but we cannot detect the opposite situation.

Reply:
I only switched it to true for testing since it won't currently pass tests with it set to false.

Reply:
And if we switch to lazy loading, then this property will disappear altogether, right?

Reply:
That would be the idea - that way it's only loaded as needed but also doesn't require any user action.